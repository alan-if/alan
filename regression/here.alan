Every obj Isa object
  Verb v
    Does
      "You v the $o."
  End Verb v.
End Every obj

Syntax l = l.
Verb l
  Does
    Look.
End Verb l.

The hero Isa actor
  Container
End The hero.

The x Isa obj At l1
End The x.

The y Isa obj At l2
End The y.

The z Isa obj In hero
End The z.

The cont Isa obj At l1
  Container
End The cont.

The a Isa obj In cont
End The a.

The l1 Isa location
End The l1.

The l2 Isa location
End The l2.

Start At l1.
	"v a (should be OK)$nv x (should be OK)$nv y (should not be OK)$nv z (should be OK)"