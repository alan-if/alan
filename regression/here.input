v a
v x
v y
v z
