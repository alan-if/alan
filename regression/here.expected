########## here ##########

        No warnings or errors detected.
        1 informational message(s).


V a (should be OK)
v x (should be OK)
v y (should not be OK)
v z (should be OK)


L1
There is a x and a cont here. The cont contains a a.

> v a
You v the a.

> v x
You v the x.

> v y
I can't see any y here.

> v z
You v the z.

> 

Do you want to UNDO, RESTART, RESTORE or QUIT ? 
